snap-overlay/bin/setup-br-ex

```diff
@@ -9,13 +9,15 @@
 
 set -ex
 
+extgateway=$(snapctl get extgateway)
+
 # Create external integration bridge
 ovs-vsctl --retry --may-exist add-br br-ex
 
 # Configure br-ex
-ip address add 10.20.20.1/24 dev br-ex || :
+ip address add $extgateway/24 dev br-ex || :
 ip link set br-ex up || :
 
-sudo iptables -t nat -A POSTROUTING -s 10.20.20.1/24 ! -d 10.20.20.1/24 -j MASQUERADE
+sudo iptables -t nat -A POSTROUTING -s $extgateway/24 ! -d $extgateway/24 -j MASQUERADE
```

Comment:
The /24 in the above feels like a big assumption that's going to bite us later - maybe we need to allow the user to provide both a network cidr and the ip address for microstack routing?

Reply:
That's a good point. Hard coding things is usually a bad idea :-)